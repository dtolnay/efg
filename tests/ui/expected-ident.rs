use efg::efg;

#[efg(unix || "false")]
struct S;

fn main() {}
