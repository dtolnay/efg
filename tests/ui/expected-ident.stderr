error: unexpected token, expected an identifier
 --> tests/ui/expected-ident.rs:3:15
  |
3 | #[efg(unix || "false")]
  |               ^^^^^^^
